### ISSUE TITLE
Pictoral font glyphs can't easily be imported

### ISSUE THREAD

**Issue by toolness** [mozilla/webmaker-addons#29]
As pictoral fonts like Font Awesome are used more and more across the web, it strikes me that people will expect the add-on to be able to import them into the sidebar, since they do _look_ like images.

**Comment by xmatthewx** [comment]
Good point. Sounds tricky. They would make for convenient stickers. File under "later"

**Comment by xmatthewx** [comment]
If we wanted to add stickers to our small feature set, we could actually link to a page that had good transparent PNGs. No need for us to create them or fit them within our space. (good answer to the question: why do this with the browser instead of a native app?)
